web3/tools/benchmark/node.py

```diff
@@ -80,20 +80,23 @@ def _geth_binary(self) -> str:
     def _geth_command_arguments(self, datadir: str) -> Sequence[str]:
         return (
             self.geth_binary,
+            "--dev",
+            "--dev.period",
+            "100",
             "--datadir",
             str(datadir),
             "--nodiscover",
-            "--fakepow",
             "--http",
             "--http.port",
             self.rpc_port,
             "--http.api",
-            "admin,eth,net,web3,personal,miner",
+            "admin,eth,net,web3",
             "--ipcdisable",
             "--allow-insecure-unlock",
```

Comment:
I think `--allow-insecure-unlock` can go as well

Reply:
No luck with this either, tests are failing without it.

Reply:
I think the calls we make to unlock account aren't necessary but I could be wrong. We might need to tackle that in a separate PR but we also might be able to use the unlocked accounts that `geth --dev` gives us. I think it's just `coinbase` but it could be others as well, would have to look at the docs.

Reply:
Let's leave it alone for now, this is beyond the scope of the PR I think